src/test/java/com/synerzip/expenseCalculation/ExpenseControllerTest.java

```diff
@@ -65,13 +72,23 @@ public void testAddExpense() throws Exception {
 		jObject.put("date", "2013-09-18");
 		jsonExpense = jObject.toString();
 
-		BDDMockito.given(expenseService.addExpense(65, expense)).willReturn(anyExpense);
+		BDDMockito.given(expenseService.addExpense(expense)).willReturn(anyExpense);
 
-		mockmvc.perform(post("/expenses/65")
+		mockmvc.perform(post("/users/expenses")
 						.contentType(MediaType.APPLICATION_JSON)
 						.content(jsonExpense))
-			   .andExpect(status().isOk());
+			   .andExpect(status().isOk())
+			   .andExpect(content().string(startsWith("{\"id\":")));
 
 	}
+	
+	@Test
+	public void testGetAll() throws Exception {
+		
+		BDDMockito.given(expenseService.getAll()).willReturn(new ArrayList<Expense>());
+		mockmvc.perform(get("/users/expenses")
+				.contentType(MediaType.APPLICATION_JSON))
+	   .andExpect(status().isOk());
```

Comment:
status code is not enough for testing any request.
you are testing get method which will return list of object so check response object.